p2p/peer_tracker_test.go

```diff
@@ -42,13 +45,17 @@ func TestPeerTracker_GC(t *testing.T) {
 
 	require.Nil(t, p.trackedPeers[pid1])
 	require.Nil(t, p.disconnectedPeers[pid3])
+
+	peers, _ := mockPeerStore.Load(context.Background())
+	require.Equal(t, pid2, p.trackedPeers[pid2].peerID)
```

Comment:
whats point of this check?

Reply:
That the peer with the best score (_i.e: the peer that didn't get deleted_) ended up in the peerstore, as we have `gc` running in the background.